contributors/devel/sig-node/cherry-picks.md

```diff
@@ -0,0 +1,115 @@
+# SIG Node Cherry-Pick Management Process
+
+Kubernetes supports multiple release versions simultaneously. How do new fixes
+make it into earlier versions, when we only develop one version at a time?
+
+We [cherry-pick] them!
+
+[cherry-pick]: https://git-scm.com/docs/git-cherry-pick
+
+## Release timeline
+
+SIG Release regularly (on a ~monthly cadence) ships [patch releases]. These are
+intended for critical bug fixes to the latest minor version, such as addressing
+security vulnerabilities, fixes to problems affecting a large number of users,
+severe problems with no workaround, and blockers for products based on
+Kubernetes.
+
+Every release, the Release Team publishes a list of [target dates] with
+**cherry-pick deadlines**. SIG Node's responsibility is to ensure that all
+patches that we want cherry-picked to previous releases must be [triaged],
+prioritized, LGTM'd, and approved by the deadline.
+
+[patch releases]: https://github.com/kubernetes/sig-release/blob/master/release-engineering/versioning.md#patch-releases
+[target dates]: https://github.com/kubernetes/sig-release/blob/master/releases/patch-releases.md#upcoming-monthly-releases
+[triaged]: triage.md
+
+## Monthly cherry-pick process
+
+Throughout the release cycle, merged bugfixes should be evaluated for inclusion
+as cherry-picks. Any bugs marked **critical-urgent** should be considered. Some
+**important-soon** bugs may also be considered if they are sufficiently
+contained and meet the criteria above. **Only** bug fixes should be considered for backports.
```

Comment:
I would propose to require some soak time for the fix in master. Maybe merged for a week without known regressions in tests

Reply:
Yes, this is true, I should add that.